src/Library/demos/Spinner/main.json

```
@@ -0,0 +1,10 @@
+{
+  "name": "Spinner",
+  "category": "user_interface",
+  "description": "Display Loading State",
```

only titles should be title cased

```suggestion
  "description": "Display loading state",
```